source/guides/non-denizen/resource-packs.md

```diff
@@ -0,0 +1,242 @@
+Resource Packs - Custom Items And Sounds
+-------------------
+
+This page will answer some common questions from programmers interested in creating visually custom items and adding new sounds into a resource pack. There are plenty of tutorials on creating your own resource pack, and less commonly how to implement custom model data and manage sounds within one. This guide primarily, details how to correctly format your resource pack, and then implement it into Minecraft using Denizen scripts.
```

Comment:
There's not a strict line length limit but please put single newlines after `.`'s where possible to keep lines not too many miles long (the actual rendered page will still merge into a paragraph)